local-volume/README.md

```diff
@@ -165,30 +165,67 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Generating Provisioner's daemonset spec and config map by using Helm chart 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option allows a user to generate Provisioner's daemon set and confgmap using **helm** templating based on predefined values,
+manual modify generated Provisioner's spec and instantiate Provisioner's daemon set and config map with **kubectl**.
+**helm template** uses 3 pieces of information:
+1. Provisioner's chart template located at helm/provisioner/templates/provisioner.yaml
+2. Provisioner's default values.yaml which contains variables used for rendering a template.
+3. (Optional) User's customized values.yaml as a part of helm template command. User's provided
+   values will override default values of Provisioner's values.yaml.
 
-1. Create an admin account with cluster admin priviledge:
+Default values.yaml is located in local-volume/helm/provisioner folder, user should not remove variables from this file but can
+change any values of these variables. 
+The only exception is the list of storage classes and respective mounting points. In general there should be at least one storage class
+listed with its corresponding "hostDir", "mountDir" parameter is optional in case and in case of absense, the value of "defaultMountDir"
+will be used.
+
+Follow these steps to bring up Provisioner and persistent local volumes.
+
+1. Install helm binary
+In order to be able to use **helm** to render templates, it has to be installed on a host where a user plans
+to generate templates.
+
+These two commands will download and copy helm binary into /usr/bin directory.
+
+``` console
+export HELM_URL=http://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz
+curl "$HELM_URL" | sudo tar --strip-components 1 -C /usr/bin linux-amd64/helm -zxf -
+```
+Provisioner's spec generation process has been tested with helm version 2.7.2.
+
+2. Create an admin account with cluster admin priviledge:
 ``` console
 $ kubectl create -f bootstrapper/deployment/kubernetes/<version>/admin-account.yaml
```

Comment:
I think it's actually provisioner/deployment/kubernetes/admin_account.yaml

Reply:
done